client/src/components/generic/Table.js

```diff
@@ -46,44 +47,66 @@ const StyledTableRow = withStyles((theme) => ({
   },
 }))(TableRow);
 
-export const Table = ({ order, orderBy, onRequestSort, columns, rows, isLoading }) => {
+export const Table = ({ order, orderBy, onRequestSort, columns, rows, isLoading, rowsPerPage = 10 }) => {
```

Comment:
I think we're expecting people to use this on desktop - how does 20 look for the default? 10 seems a little low to m but not sure how it looks in practice

Reply:
Resolved offline - 10 looks good